app/views/admin/users/index.html.erb

```diff
@@ -1,12 +1,12 @@
-<h1>Listing Users</h1>
+<h1 data-testid="listing-users">Listing Users</h1>
 
 <div class="users">
   <%= will_paginate @users %>
 
   <form class="search" method="get">
     <lable>Search</lable>
```

Comment:
I know you didn't write this line, but I just noticed this misspelled "lable" tag :) Maybe fix that while you're working on this?